test/extensions/filters/http/wasm/config_test.cc

```diff
@@ -207,6 +211,201 @@ TEST_P(WasmFilterConfigTest, YamlLoadFromRemoteWASM) {
   cb(filter_callback);
 }
 
+TEST_P(WasmFilterConfigTest, YamlLoadFromRemoteWASMFailOnUncachedThenSucceed) {
+  Envoy::Extensions::Common::Wasm::clearCodeCacheForTesting(true);
+  const std::string code = TestEnvironment::readFileToStringForTest(TestEnvironment::substitute(
```

Comment:
apparently you can use `TestEnvironment::runfilesPath(relative_path)`

Reply:
OK.  I can fix that offline.  I"ll make a note.